src/ape_accounts/accounts.py

```diff
@@ -257,3 +259,39 @@ def __decrypt_keyfile(self, passphrase: str) -> HexBytes:
             return EthAccount.decrypt(self.keyfile, passphrase)
         except ValueError as err:
             raise InvalidPasswordError() from err
+
+
+def generate_account(
+    alias: str, passphrase: str, hd_path: str = ETHEREUM_DEFAULT_PATH, word_count: int = 12
+) -> Tuple[AccountAPI, str]:
```

Comment:
I'm a little concerned about style here.  I tend to lean toward functional programming but I do wonder if style-wise, everyone would prefer to see this on `AccountManager` or something.  

I mostly just dislike extra indentation and state when it's not necessary but am happy to match style.